template/web/src/pageComponents/buy-me-coffee/MemoCard.tsx

```diff
@@ -16,17 +16,17 @@ function MemoCard({ numCoffees, twitterHandle, message, userAddress, time }: Cof
 
   return (
     <li className="flex w-full flex-col items-start gap-4">
-      <div className="items-center justify-between self-stretch lg:flex">
+      <div className="grow items-center justify-between lg:flex">
         <div className="flex items-center gap-3">
           <Avatar address={userAddress} />
-          <div className="flex items-start gap-1">
-            <span className="text-3 text-bold w-fit whitespace-nowrap font-bold text-boat-color-palette-foreground">
+          <div className="inline-flex items-start gap-1 md:flex">
+            <span className="text-3 text-bold w-fit truncate whitespace-nowrap text-wrap font-bold text-boat-color-palette-foreground">
               <Name address={userAddress} />
-              <span className="text-3 w-fit whitespace-nowrap font-normal text-boat-color-palette-foregroundmuted">
+              <span className="text-3 w-fit truncate whitespace-nowrap text-wrap break-all font-normal text-boat-color-palette-foregroundmuted">
```

Comment:
I think the Twitter Handle should be trim, and stay in just one paragraph.

Reply:
updated

<img width="768" alt="Screenshot 2024-02-17 at 12 30 22 AM" src="https://github.com/coinbase/build-onchain-apps/assets/9765685/ec80676b-29e5-424b-943d-89ee4b3a694f">

Reply:
updated (fixed the gap b/w memo name and date)
<img width="755" alt="Screenshot 2024-02-17 at 12 36 08 AM" src="https://github.com/coinbase/build-onchain-apps/assets/9765685/5c0eaf85-05c1-473e-8413-bcfb36d117a6">